### ISSUE TITLE
bertiniPosDimSolve returns wrong slice

### ISSUE THREAD

**Issue by haerski** [Macaulay2/M2#1010]
`bertiniPosDimSolve` returns a witness set whose slice doesn't contain the points
```restart
needsPackage "Bertini"

R = CC[x,y,z,t]
I = ideal(x + 3, y+1)

nv = bertiniPosDimSolve(I_*, Verbose => true)

w = first components nv
F = polySystem slice w
pts = w#Points
pts / (p -> norm evaluate(F,p)) -- this value is >> 0
```

This seems to work only when codim I < 2

```
restart
needsPackage "Bertini"

R = CC[x,y,z,t]
I = ideal(x + 3, y+1, z - 2)

nv = bertiniPosDimSolve(I_*, Verbose => true)

w = first components nv
F = polySystem slice w
pts = w#Points
pts / (p -> norm evaluate(F,p)) -- this value is almost 0
```

@JoseMath

**Comment by DanGrayson** [comment]
@antonleykin @JoseMath -- who will look at this?

**Comment by antonleykin** [comment]
This issue got resolved in version 1.16.
